x-pack/plugins/observability/public/pages/rule_details/index.tsx

```diff
@@ -0,0 +1,460 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState, useEffect } from 'react';
+import moment from 'moment';
+import { useParams } from 'react-router-dom';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiText,
+  EuiSpacer,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButtonIcon,
+  EuiPanel,
+  EuiTitle,
+  EuiHealth,
+  EuiPopover,
+  EuiHorizontalRule,
+  EuiTabbedContent,
+  EuiEmptyPrompt,
+} from '@elastic/eui';
+
+import {
+  enableRule,
+  disableRule,
+  snoozeRule,
+  unsnoozeRule,
+  deleteRules,
+} from '@kbn/triggers-actions-ui-plugin/public';
+// TODO: use a Delete modal from triggersActionUI when it's sharable
+import { DeleteModalConfirmation } from '../rules/components/delete_modal_confirmation';
+import { CenterJustifiedSpinner } from '../rules/components/center_justified_spinner';
+import { getHealthColor } from '../rules/config';
+import {
+  RuleDetailsPathParams,
+  EVENT_ERROR_LOG_TAB,
+  EVENT_LOG_LIST_TAB,
+  ALERT_LIST_TAB,
+} from './types';
+
+import { useBreadcrumbs } from '../../hooks/use_breadcrumbs';
+import { usePluginContext } from '../../hooks/use_plugin_context';
+import { useFetchRule } from '../../hooks/use_fetch_rule';
+import { RULES_BREADCRUMB_TEXT } from '../rules/translations';
+import { PageTitle, ItemTitleRuleSummary, ItemValueRuleSummary, Actions } from './components';
+import { useKibana } from '../../utils/kibana_react';
+import { useFetchLast24hAlerts } from '../../hooks/use_fetch_last24h_alerts';
+import { formatInterval } from './utils';
+import {
+  hasExecuteActionsCapability,
+  hasAllPrivilege,
+  RULES_PAGE_LINK,
+  ALERT_PAGE_LINK,
+} from './config';
+
+export function RuleDetailsPage() {
```

Comment:
nit - I think this component can break in multiple components, it will be easier for readability and unit tests